### PR TITLE
Replace repeating code for libyui press_next function by inheriting code from NavigationBase

### PR DESCRIPTION
**Pull request by sofiasyria**
- Related ticket: https://progress.opensuse.org/issues/95745
- Verification run: http://falafel.suse.cz/tests/overview?distri=sle&version=15-SP4&build=36.1

### CODE REVIEW

File: lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm [hunk 1]
```diff
@@ -630,7 +630,7 @@ sub decline_warning {
 sub show_summary_and_accept_changes {
     my ($self) = @_;
     $self->get_expert_partitioner_page()->press_next_button();
-    $self->get_summary_page()->press_next_button();
+    $self->get_summary_page()->press_next();
```
Comment (jknphy) [inline lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm:633]: we also need the replacement [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13291/commits/7d709120384b0ee966e70ad67c0414ddfaba73c8#diff-0ffd80c95fb51756da5f9d8a80e01b4f6c2a14898cf18aa594456bc79cccffcdR404) and in the line above, I think.

**Review by jknphy — COMMENTED** [review]
LGTM, nice replacement.
We could run just [this test module in this test suite](https://openqa.suse.de/tests/7150047#step/yast2_expert_partitioner/1) to ensure not missing anything.

File: lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm (hunk 1 above)
Comment (sofiasyria) [reply lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm:633]: Could you please tell me what is "here" referring to? The link point to the beginning of my second commit.

Comment (sofiasyria) [reply lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm:633]: Nevermind, I understood what you meant

Comment (jknphy) [reply lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm:633]: in line 632 you still have `press_next_button();` and the 'here' that seems I pasted wrongly was in `$self->get_add_volume_group_page()->press_next_button();` in the same file.

**Pull request merged**